drivers/ad7746/ad7746.c

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ad7746
+ * @{
+ *
+ * @file
+ * @brief       AD7746 Capacitance-to-digital converter with temperature
+ *              sensor driver
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ * @}
+ */
+
+#include "ad7746.h"
+#include "ad7746_params.h"
+#include "ad7746_internal.h"
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#include "xtimer.h"
+
+#define I2C (dev->params.i2c)
+#define ADDR (dev->params.addr)
+
+#define CONF_TEST_VALUE (1 << AD7746_CONFIGURATION_VTF1_BIT)
+
```

Comment:
Spend a comment that states what this is (conversion time[ms] for for vt channel at different sampling rates)